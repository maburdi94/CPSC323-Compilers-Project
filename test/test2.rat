
integer i   =   1;
while (i < 10) {
    put(i);
    i = i + 1;
}
