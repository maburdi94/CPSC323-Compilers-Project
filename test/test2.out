KEYWORD: integer
IDENTIFIER: i
OPERATOR: =
INTEGER: 1
SEPARATOR: ;
KEYWORD: while
SEPARATOR: (
IDENTIFIER: i
OPERATOR: <
INTEGER: 10
SEPARATOR: )
SEPARATOR: {
KEYWORD: put
SEPARATOR: (
IDENTIFIER: i
SEPARATOR: )
SEPARATOR: ;
IDENTIFIER: i
OPERATOR: =
IDENTIFIER: i
OPERATOR: +
INTEGER: 1
SEPARATOR: ;
SEPARATOR: }